### ISSUE TITLE
Zoonavigator-api can not be run as a non-root user

### ISSUE THREAD

**Issue by rajha-korithrien** [elkozmon/zoonavigator#36]
There are Kubernetes environments that provide a Pod Security Policy (PSP). One of the options in a PSP is to force all containers in a Pod to run as a non-root user (some UID/GID grater than 0).

In such an environment zoonavigator-api can not be deployed, because the docker image has everything under the ```/app``` folder owned by ```root:root``` as such the following error is encountered.
```
Oops, cannot start the server.
java.nio.file.AccessDeniedException: /app/RUNNING_PID
        at sun.nio.fs.UnixException.translateToIOException(UnixException.java:84)
        at sun.nio.fs.UnixException.rethrowAsIOException(UnixException.java:102)
        at sun.nio.fs.UnixException.rethrowAsIOException(UnixException.java:107)
        at sun.nio.fs.UnixFileSystemProvider.newByteChannel(UnixFileSystemProvider.java:214)
        at java.nio.file.spi.FileSystemProvider.newOutputStream(FileSystemProvider.java:434)
        at java.nio.file.Files.newOutputStream(Files.java:216)
        at play.core.server.ProdServerStart$.createPidFile(ProdServerStart.scala:136)
        at play.core.server.ProdServerStart$.start(ProdServerStart.scala:43)
        at play.core.server.ProdServerStart$.main(ProdServerStart.scala:25)
        at play.core.server.ProdServerStart.main(ProdServerStart.scala)
```
The design of zoonavigator-api does not require privilege or root in any way. A simple and transparent solution to this problem would be to:
* cause the ```Dockerfile``` to create a ```zoonavigator-api``` user and group in the image
* use a ```RUN``` directive to ```chown``` the ```/app``` folder to the new user
* use the ```USER``` directive to cause the ```CMD``` to be executed as the new user

There was a similar issue with zoonavigator-web where the nginx proxy required to be run with privilege (see #35) which was solved by pull request (see elkozmon/zoonavigator-web#9). Making this change will allow both zoonavigator-web and zoonavigator-api to be run in environments with restrictive security policies.

**Comment by elkozmon** [comment]
Hi! Thanks for the PR! I'll merge it and publish a new Docker image later on today.

**Comment by elkozmon** [comment]
Published with Docker tag `0.6.2` :)

**Comment by rajha-korithrien** [comment]
Works perfectly! Thanks!
